app-backend/tile/src/main/scala/StitchLayer.scala

```diff
@@ -1,87 +0,0 @@
-package com.azavea.rf.tile
-
-import com.azavea.rf.database.Database
-import com.azavea.rf.common.cache._
-import geotrellis.proj4._
-import geotrellis.raster._
-import geotrellis.vector._
-import geotrellis.vector.io._
-import geotrellis.spark._
-import geotrellis.spark.io._
-import geotrellis.spark.io.s3._
-import com.typesafe.scalalogging.LazyLogging
-import cats.data._
-import cats.implicits._
-import java.util.UUID
-
-import geotrellis.spark.io.postgres.PostgresAttributeStore
-
-import scala.concurrent._
-import scala.util.{Failure, Success, Try}
-
-object StitchLayer extends LazyLogging with Config {
-  implicit lazy val memcachedClient = LayerCache.memcachedClient
-  implicit val database = Database.DEFAULT
-
-  val system = AkkaSystem.system
-  implicit val blockingDispatcher = system.dispatchers.lookup("blocking-dispatcher")
```

Comment:
It was the case that each one of these companion objects had an instance of `blockingDispatcher`, about six of them? Now all the traits stack and use a single instance of `blockingDispatcher`, the regression is possibly from the fact that the dispatcher is now undersized for what it is doing.